pages/DetailsTrip/[city].js

```diff
@@ -0,0 +1,122 @@
+import { useRouter } from "next/router";
+import { v4 as uuidv4 } from "uuid";
+import {
+  StyleButton,
+  StyleList,
+  StyleDiv,
+  StyleCountry,
+  StyleSection,
+  StyleSpan,
+  StyleH4,
+  StyleTotalPrice,
```

Comment:
It'd be more according to convention to have it be Style**d** rather than Style, but that's something to consider for when you refactor, not now. :)